lib/sendgrid/helpers/mail/personalization.rb

```diff
@@ -63,5 +69,19 @@ def to_json(*)
         'send_at' => send_at
       }.delete_if { |_, value| value.to_s.strip == '' || value == [] || value == {} }
     end
+
+    private
+
+    def duplicate?(addition)
+      additional_email = addition.email.downcase
+
+      [@tos, @ccs, @bccs].flatten.each do |elm|
```

Comment:
SendGrid allows for duplicates across email groups. E.g., I can `to` and `bcc` the same address without issue. I don't think this library should add _more_ checks.

Reply:
According to the [docs](https://sendgrid.com/docs/for-developers/sending-email/personalizations/) SendGrid does not allow duplicates across `to`/`cc`/`bcc` within a single personalization